cw_bitcoin/lib/script_hash.dart

```diff
@@ -2,8 +2,25 @@ import 'package:bitcoin_flutter/bitcoin_flutter.dart' as bitcoin;
 import 'package:crypto/crypto.dart';
 
 String scriptHash(String address, {required bitcoin.NetworkType networkType}) {
-  final outputScript =
-      bitcoin.Address.addressToOutputScript(address, networkType);
+  try {
+    final outputScript = bitcoin.Address.addressToOutputScript(address, networkType);
+    final parts = sha256.convert(outputScript).toString().split('');
+    var res = '';
+
+    for (var i = parts.length - 1; i >= 0; i--) {
+      final char = parts[i];
+      i--;
+      final nextChar = parts[i];
+      res += nextChar;
+      res += char;
+    }
+
+    return res;
```

Comment:
what's that? and why the weird string manipulation?
if we have the parts as `abcd` then res would be `cdab` why do you need it like that?